spec/requests/assignments_spec.rb

```diff
@@ -8,6 +8,10 @@
   let(:course) { create(:course) }
   let!(:assignment) { create(:assignment, course_id: course.id) }
 
+  before(:each) do
```

Comment:
we only use before blocks because these specs were all implemented with the assumption that the authenticated user is a student. as we implement tests for teachers in these specs, we'll need to get rid of the before block and decide in each test if it should be a student or teacher that is authenticated.